src/events.js

```diff
@@ -46,7 +58,7 @@ module.exports = function(ctx) {
   };
 
   events.drawChanged = function(event) {
-    featuresChanged = event.features;
+    event.features.forEach(f => changedFeatureIds.add(f.id));
```

Comment:
Why do we need another set for `changedFeatureIds` instead of just using the store's set directly? Seems odd to me that there are two sets with similar names but they are not in sync.